docs/user/site.rst

```diff
@@ -230,6 +235,8 @@ mesh_vpn
           methods = {'salsa2012+umac'},
           -- configurable = true,
           -- syslog_level = 'warn',
+          -- delete_old_peers = false,
+          -- delete_custom_peers = false,
```

Comment:
can you elaborate on the use case of delete_custom_peers?

Reply:
This option is more about being able to retain custom peers. As the default for site values is always false I had to name it this way.
If you update from another community and your community does not support N2N or you want to abandon N2N or never planned on introducing it, you might want to get custom peers deleted also.

Reply:
what is N2N? i don't know about custom peers and N2N either, so this should be documented with this Pull Request so everyone knows what it is

Reply:
node to node VPN. I think it is great supporting that however I did not get this from the custom peers and rather would call them n2n_peers but that is just taking nit-picking to an extreme.